frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -320,6 +320,7 @@ const getActiveFilters = (
   keywordFilter,
   groupFilters,
   activeFilters,
+  categoryFilter = null,
```

Comment:
Why are we adding categoryFilter separately as it can be brought in by `filterRetentionPreference`

Reply:
I need the `categoryFilter` from the URL and only then i need to ignore saved filters. `filterRetentionPreference` only works for `kind` filter and we want it to work for kind only as of now. Just need to reset everything if user comes to the catalog with a pre-selected category.